Parse/cloud/main.js

```diff
@@ -100,6 +168,44 @@ Parse.Cloud.define("fetchUserCellData", async (request) => {
   return results;
 });
 
+Parse.Cloud.define("fetchUsersAndGroups", async (request) => {
+  console.log("Hello World");
+  var limit = request.params.limit;
+  var currentUserId = request.params.currentUserId;
+  let modelResults = [];
+
+  var userQuery = new Parse.Query("User");
+  var groupQuery = new Parse.Query("GroupDetails");
+  groupQuery.equalTo("groupAuthorUserId", currentUserId);
+  if (request.params.searchString !== undefined) {
+    userQuery.contains("searchName", request.params.searchString.toLowerCase());
+    groupQuery.contains("searchName", request.params.searchString.toLowerCase());
+  }
```

Comment:
Should we log a warning if the search string is undefined? Is that allowed here?